test/e2e/clusterctl_upgrade_test.go

```diff
@@ -67,14 +69,14 @@ var _ = Describe("When testing clusterctl upgrades using ClusterClass (CAPV 1.10
 				InitWithKubernetesVersion:                "v1.30.0",
 				WorkloadKubernetesVersion:                "v1.30.0",
 				WorkloadFlavor:                           testSpecificSettingsGetter().FlavorForMode("workload"),
-				UseKindForManagementCluster:              true,
+				UseKindForManagementCluster:              testSpecificSettingsGetter().UseKindForManagementCluster,
```

Comment:
Under which circumstances is this not true anymore?

Reply:
Currently: if `additionalVCSimServer` is set to true, this is also true.